app/models/tag.rb

```diff
@@ -10,14 +10,16 @@ class Tag < ActiveRecord::Base
 
   has_and_belongs_to_many :pull_requests
 
-  delegate :count, to: :pull_requests, prefix: true
-
   def self.with_active_pull_requests
     joins(:pull_requests).
       merge(PullRequest.active).
       uniq
   end
 
+  def pull_requests_count
+    pull_requests.where.not(status: "completed").count
```

Comment:
I should've used `PullRequest#active` for that :wink: I made the change, please, have a look.